backend/drm/meson.build

```diff
@@ -1,3 +1,7 @@
+if gbm.found() == false
```

Comment:
`not gbm.found()`

… But let's keep the GBM stuff for later

Reply:
I fix it in the other place but how could it pas my test, I did do compiling test